js-pkg/apps/demos/package.json

```diff
@@ -31,7 +33,9 @@
     "react-dom": "18.2.0",
     "tailwindcss": "^3.2.7",
     "three": "^0.149.0",
-    "typescript": "4.9.4"
+    "typescript": "4.9.4",
+    "y-codemirror": "^3.0.1",
```

Comment:
Is this unused?

Reply:
It’s used in a dynamic import:

https://github.com/drifting-in-space/driftdb/pull/56/files#diff-0d547a198d6855d31c1dc4d0ee6cd1b7e97d4ff13c46a01ada1956665592239bR68

It's a bit of a hack since CodeMirror is not SSR-friendly. Even importing it in a non-browser environment goes haywire.

Reply:
Ahhh gotcha!